scipy/optimize/tests/test_optimize.py

```diff
@@ -1284,6 +1288,17 @@ def f(x):
 
         optimize.brute(f, [(-1, 1)], Ns=3, finish=None)
 
+    def test_workers(self):
+        # check that parallel evaluation works
+        resbrute = optimize.brute(brute_func, self.rranges, args=self.params,
+                                  full_output=True, finish=None)
+
+        resbrute1 = optimize.brute(brute_func, self.rranges, args=self.params,
+                                  full_output=True, finish=None, workers=2)
+
+        assert_equal(resbrute1[-1], resbrute[-1])
```

Comment:
This may be fragile - can you use `assert_allclose` with a tolerance of a few eps?

Reply:
I can change, but shouldn't the output of a calculation be exactly the same in this case?

Reply:
As a rule I try never to compare floating point number for exact equality - it's not guaranteed in many cases (e.g. 0.0 == 0.0 isn't always true, because there are multiple binary representations of zero). It may work here, but I think it's still better to avoid.